ContactMechanics/ScanningProbe/RigidScan.py

```diff
@@ -62,4 +63,54 @@ def scan_with_rigid_sphere(topography, radius):
         scanned_heights += [
             np.max(heights[left:right] + np.sqrt(radius ** 2 - (positions[left:right] - x) ** 2) - radius)]
 
-    return np.array(scanned_heights)
+    return scanned_heights
+
+
+def pipeline_scan_with_rigid_sphere(self, radius):
```

Comment:
This directly returns a squeezed topography, which means any updated in the upstream pipeline don't automatically reflect downstream. Now
a) we don't have any use cases for this
b) it will be slow for this particular pipeline function
but the rest of the code is designed differently.

Reply:
It probably does not matter, but could we indicate the fact that a new topography is returned by the naming of the function?

Reply:
maybe add _squeeze to the end of the name and add a comment to the docstring ? 

I think that creating a "true" pipeline function for each function that returns a topography is too much work.

Reply:
By too much work, do you mean too much programming work?

Reply:
yes